src/shipping/shipping-in-store-delivery.md

```diff
@@ -0,0 +1,44 @@
+---
+title: In-store Delivery
+---
+
+The In-store delivery method allows to select a source to be used as a pickup location during the checkout of your order.
+
+![]({% link images/images/luma-in-store-example.png %}){: .zoom}
+[_Example of a In-store delivery method during checkout_]({% link sales/checkout-step1-shipping.md %})
+
+## To set up In-store Delivery
+
+The first step is to complete the default settings for In-store Delivery. You can complete this step without changing the scope of the configuration.
+
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
+
+1. In the _Sales_ section in the left panel, choose **Delivery Methods**.
+
+1. Expand ![]({% link images/images/btn-expand.png %}) the **In-Store Delivery** section.
+
+1. Set **Enabled** to `Yes`.
+
+1. If necessary, clear the **Use system value** checkbox to edit for any field to be edited.
+
+1. Enter the **Method Name** that you want to appear as a label next to the calculated rate in the shopping cart.
```

Comment:
I provided more detail earlier in this PR. Consider adapting that text for this entire procedure.

Reply:
modified, thanks!